arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -1,85 +1,127 @@
+#### 1. `chain.info`
 
-  ```
+`--chain.info-json` is a JSON string that contains required information about the Orbit chain. 
 
-  - Note for Anytrust chains, it is essential to add `--node.data-availability.enable`, and `--node.data-availability.rest-aggregator.urls=<A list of DAS REST endpoints>` or `--node.data-availability.rest-aggregator.online-url-list=<A url that returns a list of the DAS REST endpoints>` to the command or configuration.
+```shell
+--chain.info-json=<Orbit Chain's chain info>
+```
 
-- When shutting down docker image, it is important to allow for a graceful shutdown so that the current state can be saved to disk. Here is an example of how to do a graceful shutdown of all docker images currently running
+An example of `chain-info` is available in the next section.
 
-  ```shell
-  docker stop --time=300 $(docker ps -aq)
-  ```
+#### 2. `chain.name`
+
+`--chain.name` is a mandatory flag that needs to match the chain name used in `--chain.info-json`:
+
+
+```shell
+--chain.name=<My Arbitrum L3 Chain> 
+```
+
+#### 3. AnyTrust Chains
+
+For <a data-quicklook-from="arbitrum-anytrust-chain">Anytrust chains</a>, you need to to add the following flags to the command or configuration:
+
+```shell 
+--node.data-availability.enable
+```
+
+And: 
 
-## Note on permissions
+```shell 
+--node.data-availability.rest-aggregator.urls=<A list of DAS REST endpoints>
+```
 
-The Docker image is configured to run as non-root UID 1000. This means if you are running in Linux or OSX and you are getting permission errors when trying to run the docker image, run this command to allow all users to update the persistent folders
+Or: 
 
 ```shell
-mkdir /data/arbitrum
-chmod -fR 777 /data/arbitrum
+--node.data-availability.rest-aggregator.online-url-list=<A url that returns a list of the DAS REST endpoints>
 ```
 
-## Optional parameters
```

Comment:
This section is now missing, can you bring it back at the end of the page?

Reply:
Yes, it's done.